lte/gateway/python/integ_tests/s1aptests/util/traffic_util.py

```diff
@@ -298,7 +298,7 @@ class TrafficTest(object):
     _alias_lock = threading.Lock()
     _iproute = pyroute2.IPRoute()
     _net_iface = "eth2"
-    _port = 7000
+    _port = 5000
```

Comment:
why is this port changed? wont it impact other tests?

Reply:
No it wont. 5000 was the original port number. I had changed it to 7000 for ipv6 testing during the initial days of testing.